src/libs/ease/symbols.map

```diff
@@ -10,7 +10,9 @@ libelektra_0.8 {
 	elektraKeyGetRelativeName;
 	elektraKsFilter;
 	elektraResolveReference;
+};
 
+libelektra_0.9 {
```

Comment:
Why this change?

Reply:
It should have been part of #2916. The functions were not part of any 0.8.x release, so they should not be versioned as `libelektra_0.8`.

If we won't have another 0.9.x release and go straight to 1.0.0, they should actually be in `libelektra_1.0`.

Reply:
We will definitely have a 0.9 release before 1.0.